src/action.c

```diff
@@ -145,7 +145,7 @@ int main(int argc, char **argv) {
     envar_set("SINGULARITY_SHELL", singularity_registry_get("SHELL"), 1);
     envar_set("SINGULARITY_APPNAME", singularity_registry_get("APPNAME"), 1);
 
-    singularity_message(LOG, "USER=%s, IMAGE='%s', COMMAND='%s'\n", singularity_priv_getuser(), singularity_image_name(&image), singularity_registry_get("COMMAND"));
+    singularity_message(LOG, "USER=%s, IMAGE=\"%s\", COMMAND=\"%s\"\n", singularity_priv_getuser(), singularity_image_name(&image), singularity_registry_get("COMMAND"));
```

Comment:
Is there any reason not to wrap the USER string as well?

Reply:
For at least Splunk the double quotes are only necessary when the value in key=value contains a space.  It's possible a space may exist, I would assume, for IMAGE, but probably not for USER.